sky/server/requests/executor.py

```diff
@@ -520,7 +520,7 @@ async def poll_task(request_id: str) -> bool:
         raise
 
 
-def prepare_request(
+async def prepare_request(
```

Comment:
Do we need to make `add_or_update_user` async as well?

Reply:
The `create_if_not_exists` below is a request DB operation, maybe easier to make it async?